config/s3/s3api/random_size_object.yaml

```diff
@@ -0,0 +1,89 @@
+test_1:
+  TEST_ID: TEST-35678
+  object_size:
+    start : 0Kib
+    end : 100Kib
+  result_duration: 1h
+  sessions_per_node: 1
+  tool: s3api
+  operation: object
```

Comment:
Operation should have explanatory.

Reply:
Looks self-explanatory and simple. Please suggest.

Reply:
Team is okay with this so resolving this comment.